presto-main/src/main/java/io/prestosql/metadata/Split.java

```diff
@@ -62,7 +63,10 @@ public Lifespan getLifespan()
 
     public Object getInfo()
     {
-        return connectorSplit.getInfo();
+        return ImmutableMap.builder()
+                .put("catalogName", catalogName)
+                .put("connectorSplit", connectorSplit.getInfo())
```

Comment:
I think some connectors return null when we invoke `getInfo`

Reply:
yeah, that failed some tests. wrapping it in an optional now.

Reply:
will the optional be rendered the right way  in query JSON?

Reply:
this has been removed now based on Martin's suggestion.